### DISCUSSION TITLE
Marketing Team Weekly Meeting Planning 2022-03-09

### DISCUSSION THREAD

**Discussion by murillol** [WordPress/Marketing-Team#17] [General]
Sorry if this seems like a duplicate, I used the previous discussion for this week's meeting without realising the date in the title.

For next week's meeting @eidolonnight @nalininsbs:

[Agenda: Global Marketing Team, 9 March 2022]()

Join the team at 15:00 UTC in the [Marketing channel on the Make WordPress Slack](https://wordpress.slack.com/archives/C0GKJ7TFA). Async contribution is welcome too.

1. **Welcome**
2. **Getting to know each other**
    a. Attendees
    b. Social question of the week
3. **[Notes from the Last Meeting]**()
    a. Volunteer call for notetakers.
4. **Project Updates**
    a. Month in WordPress newsletter (maybe)
    b. Social Media 
    c. GitHub updates and links
    d. WCEU Contributor Day
    e. Session with Training to prepare for the 6.0 release 
    f.  People of WordPress
5. **Diary/Calendar Dates**
    Meeting time
    Follow this Google [Calendar with meetings, collaborations, and social activities](https://calendar.google.com/calendar/b/2?cid=NDQyN2g1cnZrN2EwYXA5Y3VpNDI5OGYzY3NAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ). We also list important dates in this [Forthcoming Diary Dates](https://make.wordpress.org/marketing/2021/01/13/forthcoming-diary-dates/) post. Coffee Break this week.
6. **Open Floor**

If you have any additions in mind, please comment below.

**Comment by nalininsbs** [comment]
Thank you Larissa. The aim is for more time on PofWP this week, assuming the folders are moved as hoped. I can reorganize the draft as above. 

On 6.0, reading the comments, I will aim to speak with @ bph.

**Comment by nalininsbs** [comment]
Shared in the Marketing Channel to encourage contributors to test whether they are able to add comments to this new location for the weekly discussions. Copying below for those contributors who find it easier to follow on GitHub and for asynch contributors.

Hello to everyone in the team.
We work in GitHub on planning the weekly agenda. If you have links to add for awareness or suggestions for the agenda relevant for the team or from other teams which have a potential marketing aspect, please add your comments on the ticket for this week's meeting.
We are testing whether everyone is able to add comments to this new area which is on our central repository. If you have any access difficulties, do not worry, share the details of the issue and an image so one of the team can support you. Thank you. https://github.com/WordPress/Marketing-Team/discussions/17

**Updates on tasks**
If you have an update on a task, you can update the GitHub board. If you can not find a ticket you were adding to, add a comment in this thread. This will help find any tickets that have got lost or need a revised link from the transition.
Please add any update you have on a People of WordPress story on the google document you have been assigned. Please alert us in this thread if you are having access issues to your google document. [@anjanavasan](https://wordpress.slack.com/team/U02FTSRQGF5) the document for you appears to be corrupted, and I have contacted one of the previous writers who may have taken a copy of the draft he was working on. I will come back tomorrow on this.

We have some new volunteers that we will bring on board in more detail after the main meeting this week.

**Meeting time**
We continue to meet on Wednesday at 15:00 UTC for the global team meeting. Thank you.

**Comment by DanSoschin** [comment]
Hi Folks! I'll be helping facilitate the meeting on the 9th. Just dropping a note in here to say hello.
